tests/test_nam/test_train/test_cli.py

```diff
@@ -173,22 +171,20 @@ def _setup_files(self, root_path: Path, device: _Device):
 
     def _t_main(self, device: _Device):
         """
-        End-to-end test of bin/train/main.py
+        End-to-end test of the CLI
         """
         with TemporaryDirectory() as tempdir:
             tempdir = Path(tempdir)
             self._input_path(tempdir, ensure=True)
             self._setup_files(tempdir, device)
-            check_call(
-                [
-                    "python",
-                    str(_BIN_TRAIN_MAIN_PY_PATH),
-                    str(self._data_config_path(tempdir)),
-                    str(self._model_config_path(tempdir)),
-                    str(self._learning_config_path(tempdir)),
-                    str(self._output_path(tempdir, ensure=True)),
-                    "--no-show",
-                ]
+            run(
```

Comment:
Revert; this doesn't check that the script works from the command line (and you broke it).